### ISSUE TITLE
all hosts changed in datastore by updating SANTA_DEFAULT_CLIENT_MODE in upvote/gae/shared/common/settings.py

### ISSUE THREAD

**Issue by thehesiod** [google/upvote#21]
we had an accident where I accidentally deployed from the master branch instead of our personal branch, which ended up resetting `SANTA_DEFAULT_CLIENT_MODE`.  I would have imagined this would have only affected new hosts, however ALL hosts ended up getting reset to lockdown mode triggering a lot of unhappiness :)

I think for hosts that are in the datastore should not be affected by a change to this variable.

**Comment by chief8192** [comment]
Yeah, that definitely should not happen. Sorry for the trouble this caused. Can you tell if the new deployment somehow resulted in multiple SantaHost entities being created for a single device?

**Comment by msuozzo** [comment]
Yeah this should not have happened. Looking at the code, it doesn't look like a change in this constant should have changed the settings of existing hosts. Are there other changes you've made on your branch that could have made the server think that it hadn't seen the host before?

**Comment by thehesiod** [comment]
Here are our current list of changes: https://github.com/google/upvote/compare/master...farmersbusinessnetwork:fbn  That fbn branch is going away, after it will be: https://github.com/google/upvote/compare/master...farmersbusinessnetwork:master

**Comment by thehesiod** [comment]
I don't see how any of the other changes could have caused it.  Unfortunately we wiped the DB.  I'm working on enabling backups so we can do rollbacks instead of wipes
